src/packet_sources/macos.rs

```diff
@@ -140,8 +140,9 @@ impl PacketSourceTask for MacosTask {
             }
         })
         .await?;
-        let mut from_proxy_tx =
-            pipe::OpenOptions::new().open_sender(&self.ipc_server.from_proxy_path)?;
+        let mut from_proxy_tx = pipe::OpenOptions::new()
+            .unchecked(true)
```

Comment:
What's the purpose of this? :)

Reply:
it avoids checking if the file is actually a pipe, I added it during my investigation and never removed it. The fix commit is coming